### ISSUE TITLE
Call recording not calls originated via IVR

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fusionpbx/fusionpbx#689]
```
What steps will reproduce the problem?
1. Create an IVR in GUI (apps >> IVR Menu)
2. Setup a number of IVR options that point to an extension
3. Set the extensions to Record All Calls 

What is the expected output? What do you see instead?
Inbound calls to an extension from the IVR are supposed to record but they 
don't.

Please use labels and text to provide additional information.
FusionPBX 3.6.0
Freeswitch 1.4.x
Ubunut 12.04 LTS
```

Original issue reported on code.google.com by `ewsamu...@gmail.com` on 18 Aug 2014 at 11:13

**Comment by mlungu69** [comment]
Hi

Using Version 4.0.0 this appears to still be an issue - calls passed through an IVR are not recorded.

**Comment by moteus** [comment]
I think it already fixed. https://github.com/fusionpbx/fusionpbx/commit/8c0bba7446a083e0af81fb74b5667d3d1b106883

**Comment by mlungu69** [comment]
Hi
Hmm not in in the version I have 4.0.0 - I will run an update then to check. Thanks for the input!

**Comment by moteus** [comment]
yes it fixed after release of 4.0.0

**Comment by mlungu69** [comment]
Great - thanks for that
